### DISCUSSION TITLE
Android App: Setting/Menu Option to disable all background services

### DISCUSSION THREAD

**Discussion by marek141017** [tutao/tutanota#6180] [Ideas]
Please add a menu option to disable all background services and push notifications.
There are a lot of people like me who can just open app and check for new email when they are ready.
Also your background services consume quite a lot of battery on my Pix5a with Andr14.

**Comment by nutpantz** [comment]
disable all or poll by user specified time.

**Reply by marek141017** [reply]
Hello 
What we have right now:
    We have a service in background which constantly connects to email server (or it is a socket). It means that our phones always keep connection open to email server, which is huge privacy issue because of IP addresses where our phones connect to free wifi or configured networks. This can be used to track a movements.

What I would like to propose:
    Add an ability to disable automatic email checkups (remove all back-ground services). In this mode email application will not attempt to connect to email server when application is closed or phone is locked. Emails will be checked only when one will start application and use it. This will save battery and increase privacy. 

Regards
